packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2617,6 +2616,7 @@ describe("Runtime", () => {
 
 			function createSnapshot(addMissingDatastore: boolean, setGroupId: boolean = true): void {
 				if (addMissingDatastore) {
+					// eslint-disable-next-line @fluid-internal/fluid/no-unchecked-record-access
```

Comment:
We seem to disable this all over the place (across the repo, not just in this PR). Is this entire chain of properties guaranteed by the type system to exist?

Reply:
In general no, though for various different reasons.  A lot of them are due to "split-brain" on whether the type should represent the union of the possible typing over time vs. what does it need to be today (i.e. "is a blob with this name guaranteed to exist in the snapshot").

I fixed all the cases that were trivial, and disabled the rest - but with the rule enabled for the package it will at least encourage new violations to be fixed pre-checkin.